### PR TITLE
Update changelog for 1.4.0-alpha.1

### PR DESCRIPTION
**Pull request by ahrtr**
cc @Elbehery @fuweid @ivanvc @tjungblu

### CODE REVIEW

**Comment by Elbehery** [comment]
LGTM

File: CHANGELOG/CHANGELOG-1.4.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
 
 <hr>
 
+## v1.4.0-alpha.1(TBD)
+
+### BoltDB
```
Comment (fuweid) [inline CHANGELOG/CHANGELOG-1.4.md:6]: https://github.com/etcd-io/bbolt/compare/v1.4.0-alpha.0...51926613987d57aa9df1f1bfcce0467598932e90

Do we need to mention the change about golang version?

Comment (ahrtr) [reply CHANGELOG/CHANGELOG-1.4.md:6]: updated & thx

File: CHANGELOG/CHANGELOG-1.4.md [hunk 2]
```diff
@@ -18,3 +44,6 @@
 - [Add `bbolt version` command](https://github.com/etcd-io/bbolt/pull/552)
 - [Add `bbolt inspect` command](https://github.com/etcd-io/bbolt/pull/674)
 - [Add `--no-sync` option to `bbolt compact` command](https://github.com/etcd-io/bbolt/pull/290)
+
+### Other
+- [Bump go toolchain version to 1.22.2](https://github.com/etcd-io/bbolt/pull/712)
```
Comment (fuweid) [inline CHANGELOG/CHANGELOG-1.4.md:49]: I think it belongs to 1.4.0-alpha.1

Comment (ahrtr) [reply CHANGELOG/CHANGELOG-1.4.md]: OOPS, updated & thx

**Review by fuweid — APPROVED** [review]
/lgtm

File: CHANGELOG/CHANGELOG-1.4.md [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+
+### BoltDB
+- [Enhance check functionality to support checking starting from a pageId](https://github.com/etcd-io/bbolt/pull/659)
+- [Optimize the logger performance for frequent called methods](https://github.com/etcd-io/bbolt/pull/741)
+- [Stabilize the behaviour of Prev when the cursor already points to the first element](https://github.com/etcd-io/bbolt/pull/734)
+
+### CMD
+- [Fix `bbolt keys` and `bbolt get` to prevent them from panicking when no parameter provided](https://github.com/etcd-io/bbolt/pull/682)
+- [Fix surgery freelist command in info logs](https://github.com/etcd-io/bbolt/pull/700)
+- [remove txid references in surgery meta command's comment and description](https://github.com/etcd-io/bbolt/pull/703)
```
Comment (ivanvc) [inline CHANGELOG/CHANGELOG-1.4.md:14]: Suggest to capitalize sentence to keep it consistent with the other lines:

```suggestion
- [Remove txid references in surgery meta command's comment and description](https://github.com/etcd-io/bbolt/pull/703)
```

File: CHANGELOG/CHANGELOG-1.4.md [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+- [Add rnd read capabilities to bbolt bench](https://github.com/etcd-io/bbolt/pull/711)
+- [Use `cobra.ExactArgs` to simplify the argument number check](https://github.com/etcd-io/bbolt/pull/728)
+- [Migrate `bbolt check` command to cobra style](https://github.com/etcd-io/bbolt/pull/723)
+- [Simplify the naming of cobra commands](https://github.com/etcd-io/bbolt/pull/732)
+- [Aggregate adding completed ops for read test of the `bbolt bench` command](https://github.com/etcd-io/bbolt/pull/721)
+- [add `--from-page` flag to `bbolt check` command](https://github.com/etcd-io/bbolt/pull/737)
```
Comment (ivanvc) [inline CHANGELOG/CHANGELOG-1.4.md:20]: ditto

```suggestion
- [Add `--from-page` flag to `bbolt check` command](https://github.com/etcd-io/bbolt/pull/737)
```

**Review by ivanvc — COMMENTED** [review]
Left two suggestions, other than that LGTM. Thanks, @ahrtr.

File: CHANGELOG/CHANGELOG-1.4.md (hunk 3 above)
Comment (ahrtr) [reply CHANGELOG/CHANGELOG-1.4.md]: Updated & thx

File: CHANGELOG/CHANGELOG-1.4.md (hunk 4 above)
Comment (ahrtr) [reply CHANGELOG/CHANGELOG-1.4.md]: Updated & thx

**Review by ivanvc — APPROVED** [review]
LGTM

**Pull request merged**